docker/Dockerfile

```diff
@@ -175,11 +175,6 @@ RUN apt-get update && \
     apt-get install -y --no-install-recommends \
     ca-certificates
 
-# Config settings
-
-ARG NETWORK
-ENV NETWORK=${NETWORK:-Mainnet}
```

Comment:
I think this is needed for building with `--target runtime`.

Reply:
I think so too. I just force-pushed a branch that omits this change before I read your note. Thanks for spotting it.